index.html

```diff
@@ -9,37 +9,160 @@
+                  <p class="darken">CANOPY •&nbsp;</p>
+                  <p> Back End Dev •&nbsp;</p>
+                  <p> 2015</p>
+                </div>
+                <p>A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups required.</p>
+                <ul class="card-list">
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javascipt</li>
+                </ul>
+                <button class="card-button" type="button">See Project</button>
+              </article>
+            </div>
+          </div>
+          <div class="third-card">
+            <div class="card-image"><img src="images/ThirdCover.png" alt="ThirdCover-Image"></div>
+            <div class="card-content">
+              <article>
+                <h1>Tonic</h1>
+                <div class="card-time">
+                  <p class="darken">CANOPY •&nbsp;</p>
+                  <p> Back End Dev •&nbsp;</p>
+                  <p> 2015</p>
+                </div>
+                <p>A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups required.</p>
+                <ul class="card-list">
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javascipt</li>
+                </ul>
+                <button class="card-button" type="button">See Project</button>
+              </article>
+            </div>
+          </div>
+          <div class="fourth-card">
+            <div class="card-image"><img src="images/fouthCover.png" alt="FourthCover-Image"></div>
+            <div class="card-content">
+              <article>
+                <h1>Multi-Post <br>Stories</h1>
+                <div class="card-time">
+                  <p class="darken">CANOPY •&nbsp;</p>
+                  <p> Back End Dev •&nbsp;</p>
+                  <p> 2015</p>
+                </div>
+                <p>A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups required.</p>
+                <ul class="card-list">
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javascipt</li>
+                </ul>
+                <button class="card-button" type="button">See Project</button>
+              </article>
+            </div>
+          </div>
+        </div>
+        <div class="about-section">
+          <section class="about-content">                      
+              <h1>About Myself</h1>
+              <p>
+                  My Name is Okoye Charles and I’m a software developer!
+                    I can help you build a product ,
+                  feature or website Look through some of my work and experience! 
+                  If you like what you see and need help on a project, I am available
+                    for hire.
+              </p>
+              <div class="social-header">
+                  <p class="about-sh-p">LET'S CONNECT</p>
+              </div>
```

Comment:
- [ ] [REQUIRED] Please make changes to all other titles (e.g., "Let's connect") use a header tag `hx`.

Reply:
done